projects/android/webrtc-native/.gitignore

```diff
@@ -0,0 +1 @@
+/build

```

Comment:
Can we exclude that from the `.gitignore` in `projects/` instead of having a separate file just for it?

Reply:
I can remove this .gitignore file. This build/ folder is excluded at the outer level.